src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is running");
```

Comment:
The message 'Bulldozer is running' is not very informative. Consider specifying what work the Bulldozer is doing, to make the message more informative as per the checklist.